internal/controller/operatorconfigmap_controller.go

```diff
@@ -520,6 +520,7 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI(storageClients *v1al
 			}
 		}
 		driverSpecDefaults.GenerateOMapInfo = ptr.To(c.shouldGenerateRBDOmapInfo())
+		driverSpecDefaults.EnableFencing = ptr.To(true)
```

Comment:
please address https://github.com/ceph/ceph-csi-operator/pull/281/files#r2241692992 or else this change will not work